scipy/optimize/_linprog.py

```diff
@@ -207,13 +207,17 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
         The coefficients of the linear objective function to be minimized.
     A_ub : 2-D array, optional
```

Comment:
```suggestion
    A_ub : 2-D array or sparse matrix, optional
```
Below (for `A_eq`), too.
Also, all the method-specific documentation is in `_linprog_doc.py`. Please edit this everywhere in there, too.

Reply:
You have correctly added this in all the method-specific documentation, but if it's true for every method, we would want to have it here in the top-level `linprog` documentation, too.

Reply:
My fault again. I should have read more closely/thought harder about what you were saying. Will correct if further commits necessary.